packages/api/src/external/commonwell/patient.ts

```diff
@@ -272,157 +327,109 @@ export async function update(
   });
 
   if (cwUpdateEnabled) {
-    await setPatientDiscoveryStatus({
-      patientId: patient.id,
-      cxId: patient.cxId,
+    const discoveryRequestId = requestId ?? uuidv7();
```

Comment:
ditto: requestId optional

Reply:
Not sure what you mean. You mean why is it optional in `PatientDataCommonwell` and `PatientDataCarequality`? Or why I made it optional as an input parameter?

Reply:
Referring to this https://github.com/metriport/metriport/pull/2167#discussion_r1618857625 but looks like you covered it